saleor/static/dashboard-next/siteSettings/components/SiteSettingsDetails/SiteSettingsDetails.tsx

```diff
@@ -0,0 +1,92 @@
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import TextField from "@material-ui/core/TextField";
+import * as React from "react";
+
+import CardTitle from "../../../components/CardTitle";
+import FormSpacer from "../../../components/FormSpacer";
+import i18n from "../../../i18n";
+import { SiteSettingsPageFormData } from "../SiteSettingsPage";
+
+interface SiteSettingsDetailsProps {
+  data: SiteSettingsPageFormData;
+  errors: Partial<{
+    description: string;
+    domain: string;
+    name: string;
+  }>;
+  disabled: boolean;
+  onChange: (event: React.ChangeEvent<any>) => void;
+}
+
+const SiteSettingsDetails: React.StatelessComponent<
+  SiteSettingsDetailsProps
+> = ({ data, disabled, errors, onChange }) => (
+  <Card>
+    <CardTitle
+      title={i18n.t("General Information", {
+        context: "card title"
+      })}
+    />
+    <CardContent>
+      <TextField
+        disabled={disabled}
+        error={!!errors.name}
+        fullWidth
+        name="name"
+        label={i18n.t("Store Name", {
+          context: "field label"
+        })}
+        helperText={
+          errors.name ||
+          i18n.t("Store Name is shown on taskbar in web browser", {
+            context: "help text"
+          })
+        }
+        value={data.name}
+        onChange={onChange}
+      />
+      <FormSpacer />
+      <TextField
+        disabled={disabled}
+        error={!!errors.domain}
+        fullWidth
+        name="domain"
+        label={i18n.t("Domain", {
+          context: "field label"
+        })}
+        helperText={
+          errors.domain ||
+          i18n.t("Domain is your store URL", {
```

Comment:
Why don't we give the field a better label? Currently it reads like this:

>Whatchamacallit: [_______________]
>* Whatchamacallit is your online store's full URL, eg. https://mystore.example.com/

Reply:
Maybe something like "URL of your online store"?